grails-app/controllers/au/org/ala/profile/api/ApiController.groovy

```diff
@@ -510,6 +513,177 @@ class ApiController extends BaseController {
         }
     }
 
+    @GrantAccess
+    @Path("/api/opus/{opusId}/profile/{profileId}/image/{imageId}")
+    @Operation(
+            summary = "Get images associated with a profile",
+            operationId = "/api/opus/{opusId}/profile/{profileId}/image/{imageId}",
+            method = "GET",
+            responses = [
+                    @ApiResponse(
+                            responseCode = "200",
+                            content = @Content(
+                                    mediaType = "application/json",
+                                    schema = @Schema(
+                                            implementation = ImageListResponse.class
+                                    )
+                            ),
+                            headers = [
+                                    @Header(name = 'X-Total-Count', description = "Total number of images", schema = @Schema(type = "integer")),
+                                    @Header(name = 'Access-Control-Allow-Headers', description = "CORS header", schema = @Schema(type = "String")),
+                                    @Header(name = 'Access-Control-Allow-Methods', description = "CORS header", schema = @Schema(type = "String")),
+                                    @Header(name = 'Access-Control-Allow-Origin', description = "CORS header", schema = @Schema(type = "String"))
+                            ]
+                    ),
+                    @ApiResponse(responseCode = "400",
+                            description = "opusId and profileId are required parameters"),
+                    @ApiResponse(responseCode = "403",
+                            description = "You do not have the necessary permissions to perform this action."),
+                    @ApiResponse(responseCode = "405",
+                            description = "An unexpected error has occurred while processing your request."),
+                    @ApiResponse(responseCode = "404",
+                            description = "Opus or profile not found"),
+                    @ApiResponse(responseCode = "500",
+                            description = "An unexpected error has occurred while processing your request.")
+            ],
+            parameters = [
+                    @Parameter(name = "opusId",
+                            in = ParameterIn.PATH,
+                            required = true,
+                            description = "Collection id - UUID or short name"),
+                    @Parameter(name = "profileId",
+                            in = ParameterIn.PATH,
+                            required = true,
+                            description = "Profile id - UUID or Scientific name"),
+                    @Parameter(name = "type",
+                            in = ParameterIn.PATH,
```

Comment:
ParameterIn.QUERY since it is added as query parameter and not in path.

Reply:
Changed